scripts/ci/test_plan.py

```diff
@@ -392,6 +393,9 @@ def parse_args():
             help="Number of tests per builder")
     parser.add_argument('-n', '--default-matrix', default=10, type=int,
             help="Number of tests per builder")
+    parser.add_argument('--testcase-roots-threshold', default=20, type=int,
+            help="Number of testcase roots selected for execution, base on changed files."
+                 "When exceeded, whole scope will be triggered")
```

Comment:
"Threshold value for number of modified testcase roots, up to which an optimized scope is still applied. When exceeded, full scope will be triggered"

Reply:
Thanks, changed as suggested.